application/templates/cms/edit_measure_page.html

```diff
@@ -317,15 +318,16 @@ <h3 class="heading-small">Primary source</h3>
                                         legend='Type of statistic',
                                         disabled=form_disabled)}}
 
-                  {{ render_department_select('Publisher',
+                  {{ render_department_select('Published by',
                                               form.department_source,
                                               measure.department_source_id if measure else None,
                                               organisations_by_type,
-                                              disabled=form_disabled) }}
+                                              disabled=form_disabled,
+                                              hint="For example, Ministry of Justice") }}
 
-                  {{ render_field(form.source_url, disabled=form_disabled, diffs=diffs) }}
+                  {{ render_field(form.source_url, disabled=form_disabled, diffs=diffs, hint="Link to a summary page rather than a PDF, spreadsheet or GOV.UK ‘collections page’") }}
```

Comment:
"GOV.UK collections page" feels like a relatively obscure concept (in that I don't really know what one is and I'm supposed to work with this stuff).  Will analysts (ours or elsewhere) know the difference between that and a summary page?

Reply:
Please change to: Link to a web page, not a spreadsheet or a PDF. For example, 'https://www.gov.uk/government/statistics/youth-justice-annual-statistics-2016-to-2017'

Reply:
@frankieroberto Nadia says please change this to:
"Link to a web page, not a spreadsheet or a PDF. For example, ’https://www.gov.uk/government/statistics/youth-justice-annual-statistics-2016-to-2017'"

Reply:
@Nadiahuq @TheDoubleK what should we have as the link text for that link, "Youth Justice annual statistics: 2016 to 2017"?

Also should we open in a new window (much as that usually pains me) to try and help avoid people losing the content they've not yet saved?  (In theory if you press back, it would normally still be there, but there’s still a chance it might not be).

Reply:
No link text! The URL is the example of what to type in!

Reply:
Probably shouldn't even be linked? Not sure on that one.

Reply:
@TheDoubleK eh? It's meant to be an example of the type of page to link to, not what a URL looks like...

Reply:
It shouldn't be linked! @frankieroberto and @TheDoubleK  It's an example of a URL

Reply:
We're going to use the same measure page, Youth Justice, for every bit of example text so that's it consistent across all pages and fields

Reply:
@Nadiahuq @TheDoubleK ok, I've made this change (with the unlinked URL). I'm not sure it works though. The hint duplicates the label a bit (both start with "Link to"). Also aren't we losing the message about not linking to a collection page, eg https://www.gov.uk/government/collections/youth-justice-statistics and to link to a specific publication page instead?

<img width="666" alt="screenshot 2018-11-02 at 12 08 44" src="https://user-images.githubusercontent.com/30665/47914595-15fb8300-de98-11e8-8bc0-a04e437b101f.png">

How about something like:

> Link to data source
> Please specify the URL of a web page, not a spreadsheet or PDF, and ideally make sure it is specific to this particular release.

Reply:
@TheDoubleK 😛 there's only one subclause! But yeah, it can be better worded. But does it express the general intent?  ie our preference is for a link to the page containing the specific release only, but if the only web page that exists is a page containing lots of separate releases, then that's still ok?

Reply:
Updated suggestion from @Nadiahuq:

> Source URL
> Link to a web page, not a spreadsheet or a PDF.

Reply:
@TheDoubleK I've just got a small tweak to Nadia's suggestion. Ultimately, the source data should be easy for someone to find, whatever type of page it's published on. So I've tried to emphasise that a bit more:

Source URL
Link to a web page where users can easily find the source data. Don't link directly to a spreadsheet or a PDF.